Q3.txt

```diff
@@ -20,6 +20,12 @@ while (x<b.length){
 	}
 	x=x+1;
 }
+}
+for (x=0; x<=10; x++){
```

Comment:
@aqattan the requirement is to check if the number is divisible by 10 then you loop from (0..to_the_number), also if the number is even it can be divisible by 10 at the same time so we can save extra loop right?